python/tests/test_bindings.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import hyperonpy as hp
+from hyperon import *
+from copy import deepcopy
+
+
+class BindingsTest(unittest.TestCase):
+
+    def setUp(self):
+        self.emptyBindings = Bindings()
+
+        self.bindings = Bindings()
+        self.bindings.add_var_bindings("a", SymbolAtom(hp.atom_sym("b")))
```

Comment:
1. There is a shortcut for `SymbolAtom(hp.atom_sym("b"))`: `S("b")`, it is better use it for better readability.
2. `hyperonpy` or `hp` is not intended to be used from the user code, usually there is a Python wrapper to call corresponding function. Low level functions may have some non-obvious invariants.

Reply:
done